emissionscheck_alb/data.py

```diff
@@ -0,0 +1,46 @@
+def keep_pass_fail(df):

```

Comment:
once these functions are part of a larger class, we might want to start their names with an underscore, to signal that they are helper functions and not meant to be used outside of the class

Reply:
Sounds good to me. I've never actually defined classes in Python so I'd love to learn how this is done.